docs/specs/user-model.md

```diff
@@ -204,9 +220,9 @@ If the domain part of an Email login ID is internationalized, there are 2 ways t
 
 ## Authenticator
 
-Authgear supports various types of authenticator. Authenticator can be primary, secondary or both.
+Authgear supports various types of authenticator. Authenticator must either be primary or secondary.
 
-Authenticators have priorities. The first authenticator is the default authenticator in the UI.
+A primary or secondary authenticator can be set as default. The default authenticator is used first.
```

Comment:
I think we can turn "default" to another todo? Or it is implemented already in the backend?

Reply:
This issue was discussed in 2020-09-23 product meeting. As long as we allow the user to choose which authenticator to use, we do not need the concept of default authenticator.